### PR TITLE
Improve docstrings

### PR DESCRIPTION
**Pull request by tcbegley**
This PR expands and makes consistent many of the existing docstrings and adds documentation to props that were previously undocumented.

While working on this I realised that the `FormFeedback` example in the docs was overcomplicated, and simplified it. I also noticed that the `disabled` prop of `FormGroup`, the `tabs` prop of `Nav` and `inline` prop of `FormText` serve no useful purpose so I removed them.

### CODE REVIEW

File: docs/components_page/components/form/__init__.py [hunk 1]
```diff
@@ -57,7 +57,11 @@ def get_content(app):
                 "set the `valid` property. The feedback will display if its "
                 "validity matches the validity of the `Input`. For this to "
                 "work you must wrap your `Input` and `FormFeedback` in a "
-                "`FormGroup` component."
+                "`FormGroup` component. In the below example we create two "
+                "`FormFeedback` components, one valid, one not valid."
```
Comment (pbugnion) [inline docs/components_page/components/form/__init__.py]: Missing a space after `valid.`.

File: src/components/Button.js [hunk 2]
```diff
@@ -82,7 +82,7 @@ Button.propTypes = {
   active: PropTypes.bool,
 
   /**
-   * Create block button.
+   * Create block level button, one that spans the full width of its parent.
```
Comment (pbugnion) [inline src/components/Button.js]: Is there value in adding "(default False)"?

File: src/components/card/Card.js [hunk 3]
```diff
@@ -39,22 +39,23 @@ Card.propTypes = {
...
    * Card color, options: primary, secondary, success, info, warning, danger,
-   * link. Default is secondary
+   * light, dark. Default is light.
    */
   color: PropTypes.string,
 
   /**
-   * Wrap all children in a CardBody element (basically add some padding)
+   * Apply the `card-body` class to the card, so that there is no need to also
+   * include a CardBody component in the children of this Card.
```
Comment (pbugnion) [inline src/components/card/Card.js]: Is there value in saying: "(default False)"?

File: src/components/input/Input.js [hunk 4]
```diff
@@ -165,20 +165,71 @@ Input.propTypes = {
...
 
+  /**
+   * The value of the Input
+   */
   value: PropTypes.string,
+
+  /**
+   * The initial size of the control. This value is in pixels unless the value
+   * of the type attribute is text or password, in which case it is an integer
+   * number of characters. Starting in, this attribute applies only when the
```
Comment (pbugnion) [inline src/components/input/Input.js]: That sentence is a bit strange. Starting in what?

File: src/components/input/Input.js [hunk 5]
```diff
@@ -165,20 +165,71 @@ Input.propTypes = {
...
+   * type attribute is set to text, search, tel, url, email, or password,
+   * otherwise it is ignored. In addition, the size must be greater than zero.
+   * If you do not specify a size, a default value of 20 is used.' simply
```
Comment (pbugnion) [inline src/components/input/Input.js]: It looks like something's gone wrong with `simply` here?

File: src/components/input/Input.js (hunk 4 above)
Comment (tcbegley) [reply src/components/input/Input.js]: Whoops, my bad. I copied most of these from the [dash-core-components Input](https://github.com/plotly/dash-core-components/blob/master/src/components/Input.react.js) and didn't check them that carefully. You're right this doesn't make a whole lot of sense.

**Pull request merged**